src/main/java/codeu/model/data/Conversation.java

```diff
@@ -17,48 +17,60 @@
 import java.time.Instant;
 import java.util.UUID;
 
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.UserStore;
+
 /**
- * Class representing a conversation, which can be thought of as a chat room. Conversations are
- * created by a User and contain Messages.
+ * Class representing a conversation, which can be thought of as a chat room.
+ * Conversations are created by a User and contain Messages.
  */
-public class Conversation {
-  public final UUID id;
-  public final UUID owner;
-  public final Instant creation;
-  public final String title;
+public class Conversation extends Event {
+	// public final UUID id;
```

Comment:
It seems like you used tabs for indentation instead of 2 spaces like we use in the rest of the code base. I don't know what editor you're using, but if you can't figure out how to set tabs to 2 spaces, you should be able to fix it using the find and replace function.
Tabs sometimes render differently in different editors, so it's generally better to use spaces so we know it will look the same no matter what.

Reply:
Thanks, I was having trouble with that!